docs/src/documentation/05-development/04-migration-guides/01-v16.mdx

```diff
@@ -34,6 +34,20 @@ Because of that, all occurrences of `Heading` with `type="title1"` **must be rep
 <Heading type="title0">Title 0</Heading>
 ```
 
+The Tailwind classes should also be updated. So if you are using tailwind classes that refer to `title1` in your components, be sure to update them to use `title0` in order to keep the same styles.
+
+**Before:**
+
+```jsx
+<p className="leading-heading-title1 text-heading-title1 font-heading-title1">Title 1</p>
```

Comment:
Are we considering these on the codemod? And should we? 💭

Reply:
We're not, but I guess we can 🤔 will add

Reply:
It even detected one place where the change was needed, so great! Thank you